diff --git a/node_modules/@11ty/eleventy/package.json b/node_modules/@11ty/eleventy/package.json
index 26fb67c..9d7ffc1 100755
--- a/node_modules/@11ty/eleventy/package.json
+++ b/node_modules/@11ty/eleventy/package.json
@@ -110,7 +110,7 @@
     "handlebars": "^4.7.7",
     "is-glob": "^4.0.3",
     "kleur": "^4.1.4 ",
-    "liquidjs": "^9.36.1",
+    "liquidjs": "^6.4.3",
     "lodash": "^4.17.21",
     "luxon": "^2.3.2",
     "markdown-it": "^12.3.2",
diff --git a/node_modules/@11ty/eleventy/src/Engines/Liquid.js b/node_modules/@11ty/eleventy/src/Engines/Liquid.js
index cc26c25..9ba5b8b 100644
--- a/node_modules/@11ty/eleventy/src/Engines/Liquid.js
+++ b/node_modules/@11ty/eleventy/src/Engines/Liquid.js
@@ -1,5 +1,6 @@
 const moo = require("moo");
-const liquidLib = require("liquidjs");
+// Force patch version to liquidjs@6.4.3
+const LiquidLib = require("../../../../liquidjs");
 const { TemplatePath } = require("@11ty/eleventy-utils");
 
 const TemplateEngine = require("./TemplateEngine");
@@ -27,7 +28,7 @@ class Liquid extends TemplateEngine {
 
   setLibrary(override) {
     // warning, the include syntax supported here does not exactly match what Jekyll uses.
-    this.liquidLib = override || new liquidLib.Liquid(this.getLiquidOptions());
+    this.liquidLib = override || LiquidLib(this.getLiquidOptions());
     this.setEngineLib(this.liquidLib);
 
     this.addFilters(this.config.liquidFilters);
